pkg/http/api.go

```diff
@@ -159,11 +167,28 @@ func DownloadSecrets(host string, verifyTLS bool, apiKey string, project string,
 }
 
 // GetSecrets for specified project and config
-func GetSecrets(host string, verifyTLS bool, apiKey string, project string, config string) ([]byte, Error) {
+func GetSecrets(host string, verifyTLS bool, apiKey string, project string, config string, secrets []string, includeDynamicSecrets bool, dynamicSecretsExpire time.Duration) ([]byte, Error) {
 	var params []queryParam
 	params = append(params, queryParam{Key: "project", Value: project})
 	params = append(params, queryParam{Key: "config", Value: config})
 
+	if secrets != nil {
+		params = append(params, queryParam{Key: "secrets", Value: strings.Join(secrets, ",")})
+	}
+
+	var includeDynamicSecretsOption string
+	if includeDynamicSecrets {
+		includeDynamicSecretsOption = "true"
```

Comment:
Can we use booleans here or is the string intentional?

Reply:
It just needs to be string because it's a query param (GET request)